env.c

```diff
@@ -73,7 +73,8 @@ char	*get_var_value(char **env, char *var)
 	{
 		line = ft_strnstr(env[i], var, len);
 		if (line && line[len] == '=')
-			line = line + len + 1;
+			return (line = line + len + 1);
```

Comment:
this does not return a pointer or does it? there is an assignment inside a return value which Im not sure thats NORME compatible

Reply:
yes